wpeverest/everest-forms
# Dev - Feature api log table

### All Submissions:


### Changes proposed in this Pull Request:
This pr introduces the new api logs table where we can get the detail info of the api.
To track the api log we have run the action just after api response
`do_action( 'evf_track_api_logs', $form_id, $entry_id, 'source_type', $body, $raw_response );`
Api tracking is added for the brevo, salesflare, and getResponse for now. Our team will add the that track action soon for the all integration and api calls.

### How to test the changes in this Pull Request:

1. Switch into the branch in free and pro. Run `npm i -f && npm run dev`
2. Go to tool menu > api log tabs
3. Integrate the any above mentioned integration in the form and submit the form.
4. Verify the that integration api log from the api log table.

### Types of changes:


### Other information:


### Changelog entry

> Dev - Api log table.
